src/app/wallet/view.nim

```diff
@@ -142,6 +145,26 @@ QtObject:
     write = setCurrentAccountByIndex
     notify = currentAccountChanged
 
+  proc focusedAccountChanged*(self: WalletView) {.signal.}
+
+  proc setFocusedAccountByAddress*(self: WalletView, address: string) {.slot.} =
+    if(self.accounts.rowCount() == 0): return
+
+    let index = self.accounts.getAccountindexByAddress(address)
+    if index == -1: return
+    let selectedAccount = self.accounts.getAccount(index)
+    if self.focusedAccount.address == selectedAccount.address: return
+    self.focusedAccount.setAccountItem(selectedAccount)
+    self.focusedAccountChanged()
+
+  proc getFocusedAccount*(self: WalletView): QVariant {.slot.} =
+    result = newQVariant(self.focusedAccount)
+
+  QtProperty[QVariant] focusedAccount:
+    read = getFocusedAccount
+    write = setFocusedAccountByAddress
+    notify = focusedAccountChanged
+
```

Comment:
What do you think about replacing all of this with an account lookup by address? See other comment on the QML side.